www/views/dashboard.html

```diff
@@ -13,20 +13,22 @@
           <div class="height-100">
             <div class="clearfix user-window-top">
               <div class="user-avatar fleft height-100">
-                <img src="../img/jane-doe.png" class="height-100" alt="profile image"/>
+                <img ng-src="{{dataHolder.window.model.image === null ? 
```

Comment:
cred ca logica asta de aici ar trebui mutata in controller stii daca vrem apoi sa schimbam ceva sa fie clar